recipe/meta.yaml

```
@@ -11,23 +11,25 @@ source:
   sha256: 6a7884504c6b5d5a7c3fd20356672365e00090cb66bf3bfe60b80097d1593946
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
     - dask-cuda-worker = dask_cuda.cli.dask_cuda_worker:go
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.8,<3.10
     - pip
   run:
-    - python >=3.7
+    - python >=3.8,<3.10
     - dask >=2021.11.1,<=2022.01.0
     - distributed >=2021.11.1,<=2022.01.0
     - numba >=0.53.1
     - numpy >=1.16.0
     - pynvml >=11.0.0
+    - click =8.0.4
+    - __unix
```

Think about this more as `__unix` includes macOS, we might want to go with `__linux` to restrict it further.

```suggestion
    - __linux
```